formbricks/formbricks
# feat: add ability to copy surveys between different environments of different products

## What does this PR do?

This PR allows copying surveys between different environments of different products. The issue that is fixed is #2630. No dependencies were installed.


Fixes # (issue)

<img width="1463" alt="Screenshot 2024-06-29 at 4 54 49 PM" src="https://github.com/formbricks/formbricks/assets/109029101/e5885c87-c50c-451f-8845-fb3f728c97b7">
<img width="1425" alt="Screenshot 2024-06-29 at 4 55 19 PM" src="https://github.com/formbricks/formbricks/assets/109029101/aec2ed67-921a-428f-be5c-5a958fd613cf">
<img width="803" alt="Screenshot 2024-06-29 at 4 55 36 PM" src="https://github.com/formbricks/formbricks/assets/109029101/f55939a0-8b4f-4c58-a221-75175553df4d">
<img width="1456" alt="Screenshot 2024-06-29 at 5 20 52 PM" src="https://github.com/formbricks/formbricks/assets/109029101/4fa6a159-dfbd-457c-ba2e-03443c331dfb">
<img width="1461" alt="Screenshot 2024-06-29 at 5 21 16 PM" src="https://github.com/formbricks/formbricks/assets/109029101/b0c2ec5e-75ed-4f92-855b-24ecaa17fb0e">






## How should this be tested?



- Test A: 
1.Create a survey.
2.Save changes.
3.On the dashboard, click on the copy... from the dropdown.
4.Choose the environment from different products where you want to copy.
5.Click on copy survey.
Output: Survey will be copied to the environment based upon user choices.

- Test B:
1.Create multiple surveys in different environments.
2.Select multiple environments to copy the surveys to.
3.Click the copy survey button.
Output: Verify that all selected surveys are copied correctly to the chosen environments without any duplication or errors.

## Checklist



### Required


### Appreciated
